CriteoPublisherSdk/Tests/UnitTests/Standalone/CRBannerViewTests.m

```diff
@@ -32,123 +32,75 @@
 #import "NSURL+Criteo.h"
 #import "CR_TokenValue+Testing.h"
 #import "XCTestCase+Criteo.h"
+#import "CR_DependencyProvider.h"
```

Comment:
This import is not necessary as it comes from the one after, as it is generally the case with extensions:
https://github.com/criteo/ios-publisher-sdk/blob/60713fe8d1784550afb2333b1841a0854ddbb0f7/CriteoPublisherSdk/Tests/Utility/Categories/CR_DependencyProvider%2BTesting.h#L20